features/329-identity-apis/api.md

```diff
@@ -51,7 +51,7 @@ type Identity = PasswordIdentity | OAuthIdentity | CustomTokenIdentity;
 
 async function listIdentities(): Promise<Identity[]>;
 
-async function addLoginID(loginID: { [key: string]: string }, realm?: string): Promise<void>;
+async function addLoginID(loginID: { [key: string]: string }[], realm?: string): Promise<void>;
```

Comment:
It appears much more natural to me with `{ "a": "b", "c": "d" }` instead of `[{ "a": "b"}, { "c": "d" }]`

Reply:
It is possible to have multiple login IDs with the same key, so list of login IDs is used instead.